### PR TITLE
Feature/overworld UI UX

### PR DESCRIPTION
**Pull request by 0xSense**
Top Notes:
-Player sprite directionally moving with
-Player HealthBar - UI

### CODE REVIEW

File: assets/asset/Player.png.import [hunk 1]
```diff
@@ -3,15 +3,15 @@
...
+path="res://.godot/imported/Player.png-833f18bdce549377c0230f8fa0b76e96.ctex"
 metadata={
 "vram_texture": false
 }
 
 [deps]
 
-source_file="res://scenes/combat/asset/Player.png"
-dest_files=["res://.godot/imported/Player.png-8c45d00b25ed1c2ee3b4c902405c7bf6.ctex"]
+source_file="res://assets/asset/Player.png"
```
Comment (Spencer-Trivette) [inline assets/asset/Player.png.import:13]: Good move into the assets folder, took me a while to find it deep in the scenes folder

**Review by Spencer-Trivette — COMMENTED** [review]
Good changes and solid file nesting. Do you think we should expand the assets folder for different game instances?

**Review by Spencer-Trivette — APPROVED** [review]

**Comment by 0xSense** [comment]
> Good changes and solid file nesting. Do you think we should expand the assets folder for different game instances?

What kind of structure implementation were you thinking?

**Comment by Spencer-Trivette** [comment]
> > Good changes and solid file nesting. Do you think we should expand the assets folder for different game instances?
> 
> What kind of structure implementation were you thinking?

Maybe having the combat and overworld assets set in separate files

**Pull request merged**